data/35e/wizards_of_the_coast/supplement/savage_species/ss_classes.lst

```diff
@@ -805,6 +863,39 @@ CLASS:Vampire	WEAPONBONUS:Rapier|TYPE=Martial
 #3																	BONUS:COMBAT|AC|1|TYPE=NaturalArmor.REPLACE						BONUS:STAT|DEX,CON|2
 #4						SAB:Attune fiddle (Su)								BONUS:COMBAT|AC|1|TYPE=NaturalArmor.REPLACE						BONUS:STAT|CHA|2					TEMPLATE:Grig 2
 #
+
```

Comment:
Mate, I am sure this is the easier method to implement this creature, but I think a discussion of keeping appearances uniform for the end user should be pursued. For example, this is a FEY race, which means they should be gaining FEY class (also known as HD). There should be a switch that we simply toggle that grants the changes depending on the choice.  It is more work, but the output would be much cleaner.

Reply:
You can still create a Grig using the monster manual style (using Fey), this is merely coding up the Savage Species alternative method.